build_tools/build_detect_platform

```diff
@@ -53,11 +53,13 @@ if [ -z "$ROCKSDB_NO_FBCODE" -a -d /mnt/gvfs/third-party ]; then
     FBCODE_BUILD="true"
     # If we're compiling with TSAN we need pic build
     PIC_BUILD=$COMPILE_WITH_TSAN
-    if [ -z "$ROCKSDB_FBCODE_BUILD_WITH_481" ]; then
-      source "$PWD/build_tools/fbcode_config.sh"
-    else
+    if [ -n "$ROCKSDB_FBCODE_BUILD_WITH_481" ]; then
       # we need this to build with MySQL. Don't use for other purposes.
       source "$PWD/build_tools/fbcode_config4.8.1.sh"
+    elif [ -n "$ROCKSDB_FBCODE_BUILD_WITH_7" ]; then
```

Comment:
How about changing it to ROCKSDB_FBCODE_BUILD_WITH_**PLATFORM007** instead? To me `ROCKSDB_FBCODE_BUILD_WITH_7` seems to indicate that it would use gcc-7. 

"7" and "8" is quite confusing, imo, and also since one of the internal comment on `platform007` naming mentions ...
```
gcc/glibc were removed from the name as that's confusing when clang used etc. Also it will give us more flexibility to upgrade compiler version within a platform.
```

Otherwise looks great to me.

Reply:
Well, I guess if you decide to follow through on this, it might be good to change the dependencies7 and fbcode_config7 filenames as well, so that they are all consistent.

Reply:
thanks for the idea, I will clean it up as you suggested.